Code/GraphMol/Atom.cpp

```diff
@@ -877,6 +877,27 @@ std::string getSupplementalSmilesLabel(const Atom *atom) {
   return label;
 }
 
+unsigned int numPiElectrons(const Atom *atom) {
```

Comment:
How about taking `const Atom &` instead of a pointer?

Reply:
sounds good